src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
 
+  <sky-tabset id="content-tabs">
+    
+    <sky-tab tabHeading="Design">
+      
+    <sky-demo-page-content sectionHeading="Usage">
+        
+      <h3 class="sky-text-success">Use when</h3>
+        
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>Use an error to communicate system status to users in circumstances when the system encounters an unexpected error.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/error/error-usage-1.png" />
+                <div class="caption caption-success sky-text-success"> Do use an error when a system error occurs.</div>
+            </div>
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+        <h3 class="sky-text-danger">Don't use when</h3>
+        
+        <sky-docs-visual-example>
+            
+            <sky-docs-visual-example-text>
+              <p>Don’t use an error when there is an error with a background process. Information about background processes should use <a routerLink="/components/toast">toast</a> instead.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div class="sky-margin-stacked-content-section">
+                  <img src="~/assets/img/guidelines/error/error-usage-2.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use error for failed background processes.</div>
+                </div>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-usage-7.png" />
+                    <div class="caption caption-success sky-text-success">Do use a toast to inform the user about a process operating in the background.</div>
+                </div>
+            </sky-docs-visual-example-image>
+
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Don’t use an error when the status of specific content needs user attention, but the system has not necessarily broken. To notify the user of a broken status on a workflow or record, use an <a routerLink="/components/alert">alert</a> instead.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-content-section">
+              <img src="~/assets/img/guidelines/error/error-usage-3.png" />
+                <div class="caption caption-danger sky-text-danger">Don't use the error component to represent issuse with content that user would expect to address.</div>
+            </div>
+              <div>
+              <img src="~/assets/img/guidelines/error/error-usage-4.png" />
+                <div class="caption caption-success sky-text-success">Do use alert to call attention to issues that are intended to prompt user interaction.</div>
```

Comment:
I'd change "alert" to "alerts." Also, should "alerts" be a link to that component topic? ... It's nitpicky, but the "issues" aren't really "intended to prompt user interaction." I'd change "are intended to prompt" to "require."